docs/columns/speech_order.md

```diff
@@ -1,5 +1,5 @@
 {% docs speech_order %}
 
-'Speech Order' denotes the sequence or order in which the speech was delivered during the sitting.
+Denotes the sequence or order in which the speech was delivered during the sitting.
```

Comment:
nitpick: Simply write this as "Order in which the speech was delivered during the sitting.". Everything else is redundant